### PR TITLE
Open in LiveTL from embed view button

### LINKED ISSUE #221: No way to open stream in LiveTL after using Embedded TLs
If you click on 'Embed TLs', there is no way to open the full LiveTL view without reloading the page. This creates a 'dead end' trap where there is no easy way to change between embedded/ full view unless you refresh the page or hit the browser back button.

The UX from the start should be more intuitive to a first time user, who may not know what 'Embed TLs' and 'Open LiveTL' even do.

image

### PR DESCRIPTION
**Pull request by r2dev2**
Adds this button to open in the main LiveTL ui from embed mode
open in ui button

Fixes #221

### CODE REVIEW

**Review by KentoNishi — APPROVED** [review]

File: src/js/content_scripts/injector.js [hunk 1]
```diff
@@ -105,6 +105,11 @@ const constructParams = () => {
   return params;
 };
 
+const openLiveTL = () => {
+  window.top.location =
+    chrome.runtime.getURL(`watch.html?${constructParams()}`);
+};
```
Comment (ChrRubin) [inline src/js/content_scripts/injector.js:111]: Couldn't we just export this in `utils.js` and import it whenever its needed instead of doing postMessage?

Comment (r2dev2) [reply src/js/content_scripts/injector.js:111]: We'd also need to extract `constructParams()`, but extracting should be fine.

**Comment by RachitaBas** [comment]
it works

**Comment by ChrRubin** [comment]
@RachitaBas Thanks for the feedback. However, we are currently in the middle of a framework migration, so this has been migrated and implemented on the `smelte` branch instead.

@r2dev2 I'm closing this PR in favour of 6e065ee.

**Pull request closed without merging**